<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="8" passed="7">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="105269" started-at="2019-03-21T05:27:07Z" finished-at="2019-03-21T05:28:52Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="105269" started-at="2019-03-21T05:27:07Z" finished-at="2019-03-21T05:28:52Z">
      <class name="com.training.sanity.tests.RETC_039_TEST">
        <test-method status="PASS" signature="bfrSuit()[pri:0, instance:com.training.sanity.tests.RETC_039_TEST@41eaa2]" name="bfrSuit" is-config="true" duration-ms="29" started-at="2019-03-21T05:27:07Z" finished-at="2019-03-21T05:27:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bfrSuit -->
        <test-method status="PASS" signature="bftTest()[pri:0, instance:com.training.sanity.tests.RETC_039_TEST@41eaa2]" name="bftTest" is-config="true" duration-ms="37348" started-at="2019-03-21T05:27:07Z" finished-at="2019-03-21T05:27:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bftTest -->
        <test-method status="PASS" signature="clkOnPosts()[pri:1, instance:com.training.sanity.tests.RETC_039_TEST@41eaa2]" name="clkOnPosts" duration-ms="3185" started-at="2019-03-21T05:27:44Z" finished-at="2019-03-21T05:27:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clkOnPosts -->
        <test-method status="PASS" signature="clkOnCategory()[pri:2, instance:com.training.sanity.tests.RETC_039_TEST@41eaa2]" name="clkOnCategory" duration-ms="5317" started-at="2019-03-21T05:27:47Z" finished-at="2019-03-21T05:27:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clkOnCategory -->
        <test-method status="PASS" signature="enterCredential()[pri:3, instance:com.training.sanity.tests.RETC_039_TEST@41eaa2]" name="enterCredential" duration-ms="6833" started-at="2019-03-21T05:27:53Z" finished-at="2019-03-21T05:27:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterCredential -->
        <test-method status="PASS" signature="clkOnAllPosts()[pri:4, instance:com.training.sanity.tests.RETC_039_TEST@41eaa2]" name="clkOnAllPosts" duration-ms="1967" started-at="2019-03-21T05:27:59Z" finished-at="2019-03-21T05:28:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clkOnAllPosts -->
        <test-method status="PASS" signature="clkOnAddNewBtn()[pri:5, instance:com.training.sanity.tests.RETC_039_TEST@41eaa2]" name="clkOnAddNewBtn" duration-ms="8279" started-at="2019-03-21T05:28:01Z" finished-at="2019-03-21T05:28:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clkOnAddNewBtn -->
        <test-method status="PASS" signature="actionInAddNewPost()[pri:6, instance:com.training.sanity.tests.RETC_039_TEST@41eaa2]" name="actionInAddNewPost" duration-ms="31781" started-at="2019-03-21T05:28:10Z" finished-at="2019-03-21T05:28:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- actionInAddNewPost -->
        <test-method status="FAIL" signature="clkOnChkBox()[pri:7, instance:com.training.sanity.tests.RETC_039_TEST@41eaa2]" name="clkOnChkBox" duration-ms="159" started-at="2019-03-21T05:28:41Z" finished-at="2019-03-21T05:28:42Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.training.pom.RETC_039_POM.clickCatgChkBox(RETC_039_POM.java:218)
at com.training.sanity.tests.RETC_039_TEST.clkOnChkBox(RETC_039_TEST.java:89)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clkOnChkBox -->
        <test-method status="PASS" signature="clkOnPublish()[pri:8, instance:com.training.sanity.tests.RETC_039_TEST@41eaa2]" name="clkOnPublish" duration-ms="10349" started-at="2019-03-21T05:28:42Z" finished-at="2019-03-21T05:28:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clkOnPublish -->
      </class> <!-- com.training.sanity.tests.RETC_039_TEST -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
